com.gitlab.Murmele.UDPLogger.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.gitlab.Murmele.UDPLogger",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "UDPLogger",
+  "rename-desktop-file": "UDPLogger.desktop",
+  "rename-appdata-file": "UDPLogger.appdata.xml",
+  "rename-icon": "UDPLogger",
+  "finish-args": [
+    "--socket=wayland",
+	"--socket=fallback-x11",
+    "--filesystem=home",
+	"--share=network",
+	"--filesystem=xdg-config/kdeglobals:ro"],	
+    "modules": [
+		"modules/xlnt.json",
+		"modules/hdf5.json",
+		"modules/netcdf.json",
+		"modules/netcdf_c++4.json",	
+		{
+		"name": "UDPLogger",
+		"buildsystem": "cmake",
+		"build-commands": [
+			"desktop-file-install --dir=/app/share/applications res/linux/UDPLogger.desktop",
```

Comment:
since you seem to be upstream contributor, all of this should be installed with CMake. And with the proper naming.

Reply:
any answer to that?

Reply:
Yes I am working on it. Sorry

Reply:
it is done now